### PR TITLE
New EVA suits

### PR DESCRIPTION
**Pull request by FargothUr**
## About The Pull Request

Replaces the original EVA suit with a different suit (stats are the same)
Capture

## Why It's Good For The Game
I thought it would be cool lole

## Changelog
:cl:
add: New EVA suits
change: Old EVA suits are no longer in the storage (it remains in the code)
/:cl:

### CODE REVIEW

**Comment by MrBobCool** [comment]
wow me really like ur sprite s wow u did very good speedmerge this lol haha rofl this looks amazing here is your upvote mr

File: code/game/machinery/suit_storage_unit.dm [hunk 1]
```diff
@@ -29,8 +29,8 @@
 	var/breakout_time = 300
 
 /obj/machinery/suit_storage_unit/standard_unit
-	suit_type = /obj/item/clothing/suit/space/eva
-	helmet_type = /obj/item/clothing/head/helmet/space/eva
+	suit_type = /obj/item/clothing/suit/space/commando
+	helmet_type = /obj/item/clothing/head/helmet/space/commando
```
Comment (coldud13) [inline code/game/machinery/suit_storage_unit.dm:33]: needs //lumos change

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 2]
```diff
@@ -423,11 +423,20 @@ There are several things that need to be remembered:
...
+						if(!S.lumos_snek_tauric)
+							worn_icon = 'modular_skyrat/icons/mob/clothing/taur_naga.dmi'
+						else
+							worn_icon = 'modular_lumos/icons/mob/clothing/taur_naga.dmi'
 					if(STYLE_PAW_TAURIC)
-						worn_icon = 'icons/mob/clothing/taur_canine.dmi'
+						if(!S.lumos_paw_tauric)
+							worn_icon = 'icons/mob/clothing/taur_canine.dmi'
+						else
+							worn_icon = 'modular_lumos/icons/mob/clothing/taur_canine.dmi'
```
Comment (coldud13) [inline code/modules/mob/living/carbon/human/update_icons.dm:439]: needs to mark all of this as a lumos change too

**Review by coldud13 — CHANGES_REQUESTED** [review]
looks good to me except for commenting code changes. I'll have to testmerge this when possible

**Comment by FargothUr** [comment]
I have updated the files with comments, should be good to test

**Pull request merged**